src/rules/no-svelte-internal.ts

```diff
@@ -0,0 +1,33 @@
+import { createRule } from '../utils';
+
+export default createRule('no-svelte-internal', {
+	meta: {
+		docs: {
+			description: 'svelte/internal will be removed in Svelte 6.',
+			category: 'Best Practices',
+			// TODO Switch to recommended in the major version.
+			// recommended: true,
+			recommended: false
+		},
+		schema: [],
+		messages: {
+			unexpected: 'Importing from svelte/internal is prohibited. This will be removed in Svelte 6.'
+		},
+		type: 'problem'
+	},
+	create(context) {
+		return {
+			ImportDeclaration(node) {
```

Comment:
We should probably also check `export * from 'svelte/internal'`.

Reply:
I don't expect anyone to do that, but I think it would be even better if we also check `inport('svelte/internal')`.

Reply:
Good! updated: https://github.com/sveltejs/eslint-plugin-svelte/pull/749/commits/fca148ac1273256aea7cf656f9015d481b3580f8